.circleci/config.yml

```diff
@@ -55,7 +55,6 @@ jobs:
       - run:
           name: Update to Node v16.14.2
           command: |
```

Comment:
Any specific reason why you removed this? It would be good to use a locked version of Node so that there's consistency across prod and dev.

Reply:
I followed a suggestion from the DevOP from Circle CI. Will put it back as it did not fix my problem..